src/hal/components/mult2.comp

```diff
@@ -5,6 +5,8 @@ pin out float out "out = in0 * in1";
 function _;
 license "GPL";
 author "John Kasunich";
+see_also " invert, div2 ";
```

Comment:
This should be see_also " invert(9), div2(9) ";

Reply:
I'm curious, what's with the (9) part?

Also might I gently suggest that if we over ask for little changes that in the big picture don't really matter, we risk discouraging further pull requests. I've done this myself and thought I'd mention it.
I might just point out the preferred way - for future pull requests.

The NaN and endlesserror messages was the import part to fix (and thanks for doing it)